scrapi/base/schemas.py

```diff
@@ -55,7 +55,7 @@
     "contributors": ('//dc:creator/node()', '//dc:contributor/node()', oai_process_contributors),
     "uris": {
         "canonicalUri": ('//dc:identifier/node()', oai_extract_url),
```

Comment:
uris that appear in the canonical URI field should also appear in one of the other URI fields.